contracts/zero-ex/contracts/src/features/multiplex/MultiplexOtc.sol

```diff
@@ -65,4 +65,47 @@ abstract contract MultiplexOtc is FixinEIP712 {
             state.boughtAmount = state.boughtAmount.safeAdd(makerTokenFilledAmount);
         } catch {}
     }
+
+    function _multiHopSellOtcOrder(
+        IMultiplexFeature.MultiHopSellState memory state,
+        IMultiplexFeature.MultiHopSellParams memory params,
+        bytes memory wrappedCallData
+    ) internal {
+        // Decode the tokens[], Otc order, and signature.
+        (address[] memory tokens, LibNativeOrder.OtcOrder memory order, LibSignature.Signature memory signature) = abi.decode(
+            wrappedCallData,
+            (address[], LibNativeOrder.OtcOrder, LibSignature.Signature)
+        );
+        // Validate tokens.
+        require(
+            tokens.length >= 2 &&
+                tokens[0] == params.tokens[state.hopIndex] &&
+                tokens[tokens.length - 1] == params.tokens[state.hopIndex + 1],
+            "MultiplexOtcOrder::_multiHopSellOtcOrder/INVALID_TOKENS"
+        );
+        // Pre-emptively check if the order is expired.
+        uint64 expiry = uint64(order.expiryAndNonce >> 192);
+        if (expiry <= uint64(block.timestamp)) {
+            bytes32 orderHash = _getEIP712Hash(LibNativeOrder.getOtcOrderStructHash(order));
+            emit ExpiredOtcOrder(orderHash, order.maker, expiry);
+            return;
+        }
+
+        uint256 sellAmount = state.outputTokenAmount;
+        // Try filling the Otc order. Swallows reverts.
+        try
+            IOtcOrdersFeature(address(this))._fillOtcOrder(
+                order,
+                signature,
+                sellAmount.safeDowncastToUint128(),
+                msg.sender,
```

Comment:
This line is going to conflict with https://github.com/0xProject/protocol/pull/655

Reply:
*#665